### DISCUSSION TITLE
posts/javase/base/02-history

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#362] [Announcements]
# posts/javase/base/02-history

IT蚱蜢哥|Java|大数据|Oracle|HIVE|java,java教程,Java,mysql,rabbitmq,rabbitmq教程,javascript,设计模式,GOF23设计模式,编程,开发,教程,软件开发,互联网,spring,程序员,程序

https://itzhameng.com/posts/javase/base/02-history.html

**Comment by itzhameng** [comment]
Java正在以一年两更的速度  快速发送，大概是3月和9月发布新版本

**Reply by Mister-Hope** [reply]
Migrate to you own repo as mentioned in config file, otherwise I may block you.
